robots/panda_description/urdf/panda.urdf

```diff
@@ -82,46 +84,48 @@
                 <mesh filename="package://example-robot-data/robots/panda_description/meshes/collision/link3.stl" />
             </geometry>
         </collision>
+        <inertial>
+            <origin rpy="0 0 0" xyz="2.7518e-02 3.9252e-02 -6.6502e-02"/>
+            <mass value="3.228604"/>
+            <inertia ixx="0.037242" ixy="-0.004761" ixz="-0.011396" iyy="0.036155" iyz="-0.012805" izz="0.01083"/>
+        </inertial>
     </link>
     <joint name="panda_joint3" type="revolute">
-        <safety_controller k_position="100.0" k_velocity="40.0" soft_lower_limit="-2.8973" soft_upper_limit="2.8973" />
-        <origin rpy="1.57079632679 0 0" xyz="0 -0.316 0" />
-        <parent link="panda_link2" />
-        <child link="panda_link3" />
-        <axis xyz="0 0 1" />
-        <limit effort="87" lower="-2.9671" upper="2.9671" velocity="2.3925" />
+        <origin rpy="1.5707963267948966 0 0" xyz="0 -0.316 0"/>
+        <parent link="panda_link2"/>
+        <child link="panda_link3"/>
+        <axis xyz="0 0 1"/>
+        <limit effort="87.0" lower="-2.8973" upper="2.8973" velocity="2.175"/>
+        <safety_controller k_position="100.0" k_velocity="40.0" soft_lower_limit="-2.8973" soft_upper_limit="2.8973"/>
+        <dynamics D="1" K="7000" damping="0.003" friction="0.0" mu_coulomb="0" mu_viscous="16"/>
     </joint>
     <link name="panda_link4">
-        <inertial>
-          <mass value="0"/>
-          <origin rpy="0 0 0" xyz="0 0 0"/>
-          <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
-        </inertial>
         <visual>
             <geometry>
-                <mesh filename="package://example-robot-data/robots/panda_description/meshes/visual/link4.dae" />
+                <mesh filename="package://franka_description/meshes/visual/link4.dae"/>
```

Comment:
We should use the path relative to example-robot-data.

There are other places where we need to revert back this.